### DISCUSSION TITLE
[Feature] "fully remote" mode which doesn't read local images at all

### DISCUSSION THREAD

**Discussion by yu-jingrui** [immich-app/immich#14121] [Feature Request]
### I have searched the existing feature requests to make sure this is not a duplicate request.

- [X] Yes

### The feature

Currently there is an option to prioritize loading from remote, but I want to push it further. Due to the fact that Immich is still in active development, I haven't deployed it for doing backup. I use some synology photos to do the actual backup and use immich as an advanced viewing tool.

Therefore I would like a pure remote mode, where local photos are not loaded at all and only remote photos are shown in the timeline. This would avoid the long loading time as well as avoid showing every photo of mine twice.

### Platform

- [ ] Server
- [ ] Web
- [X] Mobile

**Comment by alextran1502** [comment]
You can already do this by not selecting any album for backup, then all you see are from the server

**Comment by yu-jingrui** [comment]
I turned yhe backup function off. I can't choose which to sync.

Alex ***@***.***> 于 2024年11月13日周三 17:23写道：
